test/Test/Data/List.purs

```diff
@@ -398,6 +407,27 @@ testList = do
   log "append should be stack-safe"
   void $ pure $ xs <> xs
 
+  log "mapReverse should work the same as slower naive version"
```

Comment:
I’d actually prefer to just test the functions we care about directly. If we have a few individual cases with inputs and expected outputs, plus one or two really big examples to test stack safety, that would be ideal I think. In my experience 100k elements is a good amount: enough to reliably cause stack overflows if there are any stack safety issues.

Reply:
Removed this test, since that function is no longer exported